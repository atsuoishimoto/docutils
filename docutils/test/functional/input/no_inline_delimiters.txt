ab*cd*ef

ab\*cd*e*f

ab*cd\*e*f

ab**cd**ef

abc_def

abc\_def

.. no-inline-delimiters:: no

ab*cd*ef

ab\*cd*e*f

.. no-inline-delimiters:: yes

ab*cd*ef

ab\*cd*e*f

.. no-inline-delimiters:: NO
