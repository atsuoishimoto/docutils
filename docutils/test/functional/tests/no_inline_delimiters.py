# Source and destination file names.
test_source = "no_inline_delimiters.txt"
test_destination = "no_inline_delimiters.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html"

# Settings
# test for encoded attribute value:
settings_overrides['no_inline_delimiters'] = True
# local copy of default stylesheet:
settings_overrides['stylesheet_path'] = ( 
            'functional/input/data/html4css1.css')

