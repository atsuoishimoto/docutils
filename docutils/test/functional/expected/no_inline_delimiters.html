<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
<title></title>
<link rel="stylesheet" href="../input/data/html4css1.css" type="text/css" />
</head>
<body>
<div class="document">


<p>ab<em>cd</em>ef</p>
<p>ab*cd<em>e</em>f</p>
<p>ab<em>cd*e</em>f</p>
<p>ab<strong>cd</strong>ef</p>
<p><a href="#id1"><span class="problematic" id="id2">abc_</span></a>def</p>
<p>abc_def</p>
<p>ab*cd*ef</p>
<p>ab*cd*e*f</p>
<p>ab<em>cd</em>ef</p>
<p>ab*cd<em>e</em>f</p>
<div class="system-message">
<p class="system-message-title">System Message: ERROR/3 (<tt class="docutils">functional/input/no_inline_delimiters.txt</tt>, line 25)</p>
<p>Value for &quot;no-inline-delimiters&quot; directive should be &quot;yes&quot; or &quot;no&quot;: &quot;NO&quot;</p>
<pre class="literal-block">
.. no-inline-delimiters:: NO
</pre>
</div>
<div class="system-messages section">
<h1>Docutils System Messages</h1>
<div class="system-message" id="id1">
<p class="system-message-title">System Message: ERROR/3 (<tt class="docutils">functional/input/no_inline_delimiters.txt</tt>, line 9); <em><a href="#id2">backlink</a></em></p>
Unknown target name: &quot;abc&quot;.</div>
</div>
</div>
</body>
</html>
